providers/inwx/inwxProvider.go

```diff
@@ -0,0 +1,292 @@
+package inwx
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/diff"
+	"github.com/StackExchange/dnscontrol/v3/providers"
+
+	"github.com/svenpeter42/goinwx"
+)
+
+/*
+INWX Registrar and DNS provider
+
+Info required in `creds.json`:
```

Comment:
The comment mentions `domain`

Reply:
yes, domain is used in the integration tests. i've added a comment.